src/test/objectstore/store_test.cc

```diff
@@ -9593,6 +9606,97 @@ TEST_P(StoreTest, BluestorePerPoolOmapFixOnMount)
   bstore->mount();
 }
 
+class hugepaged_raw;
+
+static bool is_hugepaged(const bufferptr& bp)
+{
+  const auto& ibp =
+    static_cast<const ceph::buffer_instrumentation::instrumented_bptr&>(bp);
+  return ibp.is_raw_marked<BlockDevice::hugepaged_raw_marker_t>();
+}
+
+TEST_P(StoreTestDeferredSetup, BluestoreHugeReads)
```

Comment:
Would this test case work properly if huge pages are disabled at the host? I presume it wouldn't....

Reply:
No, it won't. Commented on that in the PR's description.

Reply:
so this would rather fail the general QA testing. Not to mention private dev runs... IMO we should have automatic disablement of it then....

Reply:
Well, I was and still am afraid of that. Though, my dev env. offered HP support out-of-the-box and I was curious even about the nodes serving `make check`. It looks they are fine.

Reply:
Disabled by default; commented why.